cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go

```diff
@@ -27,21 +27,21 @@ limitations under the License.
 //   - Support custom environment variables in control plane components under `ClusterConfiguration`.
 //     Use `APIServer.ExtraEnvs`, `ControllerManager.ExtraEnvs`, `Scheduler.ExtraEnvs`, `Etcd.Local.ExtraEnvs`.
 //   - The ResetConfiguration API type is now supported in v1beta4. Users are able to reset a node by passing
-//     a --config file to "kubeadm reset".
-//   - `dry-run` mode in is now configurable in InitConfiguration and JoinConfiguration.
+//     a `--config` file to "kubeadm reset".
```

Comment:
should we make all `kubeadm ...` commands not use the `'` or `"` quotes and instead use the ` (back quote)

Reply:
Updated.